src/Elmish.WPF/Binding.fs

```diff
@@ -253,19 +253,53 @@ module Binding =
     { Name = binding.Name
       Data = binding.Data |> f }
 
-  let mapModel        f = f |> BindingData.mapModel        |> mapData
-  let mapMsgWithModel f = f |> BindingData.mapMsgWithModel |> mapData
-  let mapMsg          f = f |> BindingData.mapMsg          |> mapData
+  /// <summary>
+  ///   Map the model type parameter of a binding via a contravariant mapping.
+  /// </summary>
+  /// <param name="f">The mapping function.</param>
+  /// <param name="binding">The binding to be mapped.</param>
+  let mapModel (f: 'a -> 'b) (binding: Binding<'b, 'msg>) = binding |> mapData (BindingData.mapModel f)
```

Comment:
Regarding the docs you've added to these functions: I don't think Intellisense will ever show the param docs for F# functions. You can type `(` after the function name to see the first one, but I have never been able to make the rest show up. (For methods it works fine.)

I usually don't document function parameters unless there's something important to be said, and if so, integrate it into the main text.

For that reason, I also never use `<summary>`; I just do

```f#
/// Paragraph 1
///
/// Paragraph 2
let myFun = ...
```

Reply:
Ok.  Actually thought that about that, but wrote it this way anyway.  I think descriptions for the arguments are not needed anyway.

I pushed [this commit](https://github.com/elmish/Elmish.WPF/pull/256/commits/ad8e85bdae367fd12c84a387325ea73cac9ed7c0) to make this change.  I will leave this for you to resolve or comment again.